client/src/components/LoginSession/tests/LoginSession-story.js

```diff
@@ -1,7 +1,19 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import LoginSession from 'components/LoginSession/LoginSession';
+import { Component as LoginSession } from 'components/LoginSession/LoginSession';
 import { withKnobs, boolean } from '@storybook/addon-knobs/react';
+import backend from 'lib/Backend';
+
+window.ss.config = {
+    SecurityID: 1234567890
+};
+
+const endpointFetcher = async () => ({
```

Comment:
The pattern library is about look, not about interaction. You shouldn't be mocking HTTP request here.

That's part of the reason why you component should be split into:
- a UI component
- a container component that handles the HTTP interaction and redux updates